classes/driver/field/abstract.php

```diff
@@ -584,18 +520,30 @@ protected function getOption($path, $default = null)
      *
      * @return array
      */
-    protected function getOptions()
+    public function getOptions()
     {
         return $this->options;
     }
 
     /**
+     * Sets the options
+     *
+     * @param array $options
+     * @return $this
+     */
+    public function setOptions(array $options)
+    {
+        $this->options = $options;
```

Comment:
I'm not sure if `options` means `settings` or `options of a select list` here (it's a confusing term), but in the first case it could be useful to do an `$this->options = \Arr::merge($this->options, $options);` instead.